src/host/registry.cpp

```diff
@@ -269,15 +266,9 @@ void Registry::LoadFromRegistry(_In_ PCWSTR const pwszConsoleTitle)
     //      HOWEVER, the defaults might not have been auto-pos, so don't assume that they are.
 
     // Code Page
-    Status = RegistrySerialization::s_QueryValue(hTitleKey,
-                                                 CONSOLE_REGISTRY_CODEPAGE,
-                                                 sizeof(dwValue),
-                                                 REG_DWORD,
-                                                 (PBYTE)&dwValue,
-                                                 nullptr);
-    if (NT_SUCCESS(Status))
+    if (const auto codePage = loadDWORD(CONSOLE_REGISTRY_CODEPAGE))
     {
-        _pSettings->SetCodePage(dwValue);
+        _pSettings->SetCodePage(codePage.value());

```

Comment:
I was going to say "what about the exception" but I guess the `if` has got you covered on that as it will test `false`. Nice.